tests/contrib/operators/test_gcp_spanner_operator.py

```diff
@@ -164,6 +170,78 @@ def test_instance_delete_ex_if_param_missing(self, project_id, instance_id, exp_
         self.assertIn("The required parameter '{}' is empty".format(exp_msg), str(err))
         mock_hook.assert_not_called()
 
+    @mock.patch("airflow.contrib.operators.gcp_spanner_operator.CloudSpannerHook")
+    def test_instance_query(self, mock_hook):
+        mock_hook.return_value.execute_sql.return_value = None
+        op = CloudSpannerInstanceDatabaseQueryOperator(
+            project_id=PROJECT_ID,
+            instance_id=INSTANCE_ID,
+            database_id=DB_ID,
+            query=INSERT_QUERY,
+            task_id="id"
+        )
+        result = op.execute(None)
+        self.assertIsNone(result)
```

Comment:
Can you add assert_called_once on mock_hook.return_value.execute_sql ?

assert_called_once_with(...) may be even better.

Reply:
Done, sorry, not sure why it was not here